editor/plugins/ot_features_plugin.cpp

```diff
@@ -184,6 +184,7 @@ void OpenTypeFeaturesAdd::_bind_methods() {
 
 OpenTypeFeaturesAdd::OpenTypeFeaturesAdd() {
 	set_text(TTR("Add Feature..."));
+	EditorInspector::setup_inspector_action_button(this);
```

Comment:
All of them are inspector plugins, so they technically have a dependency.

Reply:
I guess a more idiomatic way would be a helper that _creates_ the button (with some text and an optional icon) and sets it up in the process before returning to the caller?

Reply:
Can't use a method that creates a button, because it won't work in OpenTypeFeaturesAdd. Setup is more universal.

Reply:
> it won't work in OpenTypeFeaturesAdd.

It will be removed in #61406, and new plugin is using almost the same code as a `Dictionary` property editors.

Reply:
Ok changed the method to create a button with text.